web/src/app/forms/FormField.js

```diff
@@ -186,14 +186,36 @@ export class FormField extends React.PureComponent {
           error={checkbox ? undefined : Boolean(props.error)}
           label={this.props.formLabel ? null : props.label}
         />
-        {!noError && (props.error || props.hint) && (
-          <FormHelperText>
-            {(props.error &&
-              props.error.message.replace(/^./, (str) => str.toUpperCase())) ||
-              props.hint}
-          </FormHelperText>
-        )}
+        {!noError &&
+          (props.error || props.hint) &&
+          this.renderFormHelperText(props.error, props.hint)}
```

Comment:
Would this not work?

Reply:
Ah, I see, unfortunate naming. Remember to leave a note for the reviewer if a change request has questions or couldn't be made for some reason -- otherwise it seems like it may have been missed.

Reply:
gotcha!